src/index_runner/main.py

```diff
@@ -108,13 +108,16 @@ def main():
                 logger.info('End of stream.')
             else:
                 logger.error(f"Kafka message error: {msg.error()}")
+            consumer.commit()
```

Comment:
Hmm, this will also commit if the error is a network partition, which may not be what you want.

Reply:
Yeah I'm seeing this does not make sense for an ERR__PARTITION_EOF. 
I may move it into the else block, but I'm still not sure that makes sense. Trying to find docs about it.

Reply:
Reverted this for now